one-ui/ui/src/util/roles.tsx

```diff
@@ -33,6 +35,12 @@ export class RolesService implements IRolesContextInterface {
     public canWriteMappings:() => boolean = () => {
         return this.roles.includes('data-hub-mapping-writer');
     };
+    public canReadMatching:() => boolean = () => {
```

Comment:
@hao1st Are we at a point where these can become capabilities so we're not exposing role names to the GUI? Or is the plan to take care of that in a separate refactoring ticket?

Reply:
@rjrudin I just created a task to remove role names as login response once the UI does not use roles any more.
https://project.marklogic.com/jira/browse/DHFPROD-4583

Reply:
Sounds good